README.md

```diff
@@ -2,6 +2,8 @@
 
 Resident Contact API will return a list of contact details (inc. mobile, uprn, email) for residents that we have stored.
```

Comment:
This feels more like a description for the LIST endpoint rather than the API

Reply:
I wanted opinions on this, do you think I should remove it, it was already there so wasn't too sure

Reply:
I think you could rephrase it to describe the API. Maybe something like:
```suggestion
Resident Contact API allows access to saved contact details (inc. mobile, uprn, email) for residents that we have stored.
```